app/assets/javascripts/common/regions_search.js

```diff
@@ -40,17 +40,21 @@ RegionsSearch = function () {
       })
       this.populateDropdown(html);
     } else {
-      let html = $("template#no-results-found").html();
-      let $html = $(html);
-      $html.find(".search-query").html(searchQuery);
-      this.populateDropdown($html)
+      this.populateDropdown(this.noResultsFound(searchQuery))
     }
   }
 
+  this.clearDropDown = () => {
+    $(".typeahead-dropdown").empty();
+  }
+
   this.searchRequest = (e) => {
     let searchQuery = e.value;
-
-    if (searchQuery.length) {
+    if (searchQuery === "") {
```

Comment:
Need to handle this case specifically for when a user uses backspace to clear out a query and start over.